### DISCUSSION TITLE
Running whisper in Python to transcribe audio seems super slow

### DISCUSSION THREAD

**Discussion by orionflame** [openai/whisper#656] [Q&A]
can delete, figured it out.

**Comment by vbalashi** [comment]
Could you please post what have you figured out? I have the same issue here

**Comment by orionflame** [comment]
I specified the language parameter and also switched to VS code instead.

**Reply by vbalashi** [reply]
That's weird)
it takes 20 minutes to complete this via CLI for a 2-minute MP3 file on my MacPro 2020 with 16GB memory:
whisper p1l21.mp3 --language Dutch --model large,

while Buzz GUI finishes the same thing in just 2 minutes?
and freesubtitles.ai even faster - in 30 seconds.

I wonder how switching to VS Code could improve the speed...

**Comment by orionflame** [comment]
I did a bunch of things but it could likely be IDLE for some reason. I was also using GPT code which had wrong function call to model.transcribe, but I switched to whisper.transcribe.
